pydust/src/pytypes.zig

```diff
@@ -599,6 +599,40 @@ fn BinaryOperator(
     };
 }
 
+fn EqualsOperator(
+    comptime definition: type,
+    comptime op: []const u8,
+) type {
+    return struct {
+        const equals = std.mem.eql(u8, op, "__eq__");
+        fn call(pyself: *ffi.PyObject, pyother: *ffi.PyObject) callconv(.C) ?*ffi.PyObject {
+            const func = @field(definition, op);
+            const typeInfo = @typeInfo(@TypeOf(func));
+            const sig = funcs.parseSignature(op, typeInfo.Fn, &.{});
```

Comment:
Isn't this supposed to mean "there are no valid self argument types"? Should this not fail? And instead have `&.{ py.PyObject, *const definition }`

Reply:
This is only in cases where there's only one argument